app/templates/components/footer-boxes.hbs

```diff
@@ -0,0 +1,18 @@
+  <div class="providerBox" style="width: 100%; height: 18em;">
```

Comment:
Once this component is in the pod structure a stylesheet can be added -- see `facet-search`. It's usually best to avoid inline styling.